qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/VerifyLoginSteps.java

```diff
@@ -3,32 +3,35 @@
  * v. 2.0. If a copy of the MPL was not distributed with this file, You can
  * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
  * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
- *
+ * 
  * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
  * graphic logo is a trademark of OpenMRS Inc.
  */
-package org.openmrs.contrib.qaframework.legacy;
+package org.openmrs.contrib.qaframework.automation;
 
+import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
 
-import org.junit.Test;
-import org.junit.experimental.categories.Category;
-import org.openmrs.contrib.qaframework.helper.BuildTests;
-import org.openmrs.contrib.qaframework.page.HomePage;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import org.openmrs.contrib.qaframework.RunTest;
 
-public class LoginTest extends ReferenceApplicationTestBase {
+public class VerifyLoginSteps extends Steps {
 
-    private HomePage homePage;
+    @Before(RunTest.HOOK.SELENIUM_DEFAULT_LOGIN)
+    public void visitHomePage() {
+        initiateWithLogin();
+    }
     
-    private void initiateHomePage() {
-        homePage = new HomePage(page);
-        assertPage(homePage.waitForPage());
+    @After(RunTest.HOOK.SELENIUM_DEFAULT_LOGIN)
+    public void destroy() {
+        quit();
     }
 
-    @Test
-    @Category(BuildTests.class)
+    @And("user verifies modules available on home page after login as admin")
     public void verifyModulesAvailableOnHomePage() {
```

Comment:
This method name needs to have the role that it is testing

Reply:
I think i will change method name.

Reply:
Did you address the above?

Reply:
Resolved. Thanks.